Fitness/Controllers/TabBarController.swift

```diff
@@ -36,3 +37,13 @@ class TabBarController: UITabBarController {
 
     }
 }
+
+extension TabBarController: UITabBarControllerDelegate {
+    override func tabBar(_ tabBar: UITabBar, didSelect item: UITabBarItem) {
+        if let tabName = item.title {
```

Comment:
you can condense to `if let tabName = item.title, tabName == "browse" {...}` #swiftIsAmazing